### DISCUSSION TITLE
Table module decouple

### DISCUSSION THREAD

**Discussion by guilhermeprokisch** [pimalaya/himalaya#355] [Ideas]
I like Himalaya table formating. Fits very nice with fzf. I would like to use this style with other programs given an arbitrary JSON for example. It's reasonable to decouple the table format in a separate module? Thanks.

**Comment by soywod** [comment]
I thought about exporting the table module, but I do not have enough
time to do so these days. I keep this issue open for the time I can
finally take care of it, or in case someone wants to do it. Thank you!

On 05 Apr 2022, at 22:47 (+0200), soywod/himalaya wrote:
> I like Himalaya table formating. Fits very nice with fzf. I would like to use this style with other programs give an arbitrary JSON for example. It's reasonable to decouple the table format in a separate module? Thanks.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/soywod/himalaya/discussions/355
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
Cordialement
Clément DOUIN
Développeur Web Full-Stack
https://soywod.me
